package.json

```diff
@@ -32,6 +32,7 @@
   },
   "homepage": "https://github.com/dignifiedquire/pull-length-prefixed#readme",
   "dependencies": {
+    "mocha": "^3.2.0",
```

Comment:
Ah, thanks for catching that, leftovers from my tests figuring out what's wrong with node 4.